ignite/src/main/java/org/hibernate/ogm/datastore/ignite/query/impl/IgniteQueryDescriptor.java

```diff
@@ -20,16 +20,19 @@
 
 	private final String sql;
 	private final List<Object> indexedParameters;
+	private final String table;
 	private final boolean hasScalar;
 //	private final List<Return> customQueryReturns;
 //	private final Set<String> querySpaces;
 
-	public IgniteQueryDescriptor(String sql, List<Object> indexedParameters, boolean hasScalar) {
+	public IgniteQueryDescriptor(String sql, String table, @Deprecated  List<Object> indexedParameters, boolean hasScalar) {
```

Comment:
Why is that value deprecated?

Reply:
Main idea is migration to native hibernate query parameters.